funcx_sdk/funcx/utils/errors.py

```diff
@@ -111,3 +111,14 @@ def __init__(self, message):
 
     def __repr__(self):
         return "Invalid Scope: {}".format(self.message)
+
+
+class TaskPending(FuncxError):
+    """Task is pending and no result is available yet
+    """
+
+    def __init__(self, reason):
+        self.reason = reason
+
+    def __repr__(self):
+        return "TaskPending due to {self.reason}"
```

Comment:
the format could be nicer, probably either `Task pending due to {self.reason}` or `Task is pending due to {self.reason}`

Reply:
Done